src/Money.php

```diff
@@ -641,6 +657,10 @@ private function gcdOfMultipleInt(array $values): int
     {
         $values = array_map(fn (int $value) => BigInteger::of($value), $values);
 
+        /**
+         * @psalm-suppress ImpureMethodCall
+         * FIXME: remove the suppression after BigInteger::gcdMultiple() is marked @psalm-pure
+         */
```

Comment:
What prevents `BigInteger::gcdMultiple()` from being marked as pure?

Reply:
this is another library